### PR TITLE
Fix refresh script for OSX

### PR DESCRIPTION
**Pull request by robb-romans**
* Detect if we're on Docker for Mac (xhyve)
* Otherwise, use docker-machine (Docker Toolbox)
* Make IP variable more descriptive
* Add check for grunt executable

### CODE REVIEW

File: script/refresh [hunk 1]
```diff
@@ -4,9 +4,19 @@ set -euo pipefail
...
+if [[ ${OSX} -eq 0 ]]; then
+    DECONST_IP="127.0.0.1"
+else
+    DOCKER_MACHINE_NAME=${DOCKER_MACHINE_NAME:-dev};
+    DECONST_IP=$(docker-machine ip ${DOCKER_MACHINE_NAME})
+fi
 
-DECONST=$(docker-machine ip ${DOCKER_MACHINE_NAME})
+if ! [[ -x "$(command -v grunt)" ]]; then
+    echo "Error: grunt command not found. Please install it."
```
Comment (nimbinatus) [inline script/refresh]: Minor note: I'd reword this error to be something like this to be a bit closer to the common shell error message. Might help folks who aren't too familiar with the terminal:
```bash
echo "Error: grunt: command not found. Install 'grunt' and retry."
```

**Review by nimbinatus — APPROVED** [review]
Just a minor note; not a blocker for merging.

**Comment by robb-romans** [comment]
@nimbinatus thanks. updated.
